scripts/watchdog

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+VERBOSE=no
+
+function debug()
+{
+    if [[ x"${VERBOSE}" == x"yes" ]]; then
+        echo "$date $@"
+    fi
+    logger "$@"
+}
+
+function disable_watchdog()
+{
```

Comment:
Please detect platform first. This change only works for Dell. You also need to do things differently for Arista/Mellanox. And preferably, generate a log message state platform watchdog support is missing otherwise.